src/detect-http-client-body.c

```diff
@@ -258,6 +258,7 @@ static InspectionBuffer *HttpClientBodyGetDataCallback(DetectEngineThreadCtx *de
     StreamingBufferGetDataAtOffset(body->sb,
             &data, &data_len, offset);
     InspectionBufferSetup(buffer, data, data_len);
+    InspectionBufferApplyTransforms(buffer, transforms);
```

Comment:
I think https://github.com/OISF/suricata/pull/5213/commits/39760336c22f043c060ac36d64d3dc04dd467e9f should be dropped for now. @jlucovsky is working on addressing to issues around applying transforms to HTTP.

Reply:
ok

Reply:
S-V test no longer passes without this commit (and this PR does not make much sense either)
@jlucovsky what is next ?